rsl-learning2022-Risheel/WIFIIndicator/app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="ShapeAppearanceOverlay.MyApp.Button.Circle" parent="">
```

Comment:
As mentioned in the assignment document, you have to use the shape drawables to draw the circular shape.